docs/demos/first_steps.md

```diff
@@ -0,0 +1,299 @@
+
+
+This document will walk you through the steps to:
+  - install Open Service Mesh (OSM) v0.8.0-rc.2
+  - install 2 sample apps: `bookbuyer` and `bookstore`
+  - apply [Service Mesh Interface (SMI)](https://smi-spec.io/) policies to control the traffic between these pods
+
+
+## Prerequisites
+This demo of OSM v0.8.0-rc.2 requires:
+  - a cluster running Kubernetes v1.15.0 or greater
+  - a workstation capable of executing [Bash](https://en.wikipedia.org/wiki/Bash_(Unix_shell)) scripts
+  - [The Kubernetes command-line tool](https://kubernetes.io/docs/tasks/tools/#kubectl) - `kubectl`
+
+
+## Download the OSM command-line tool
+
+The `osm` command-line tool contains everything needed to install and configure Open Service Mesh.
+The binary is available on the [OSM GitHub releases page](https://github.com/openservicemesh/osm/releases/).
+
+### For GNU/Linux or macOS
+Download and unzip the 64-bit [GNU/Linux](https://github.com/openservicemesh/osm/releases/download/v0.8.0-rc.2/osm-v0.8.0-rc.2-linux-amd64.tar.gz)
+or
+[macOS](https://github.com/openservicemesh/osm/releases/download/v0.8.0-rc.2/osm-v0.8.0-rc.2-darwin-amd64.tar.gz)
+binary of OSM v0.8.0-rc.2:
+```bash
+system=$(uname -s | tr '[:upper:]' '[:lower:]')
+release=v0.8.0-rc.2
+curl -L https://github.com/openservicemesh/osm/releases/download/${release}/osm-${release}-${system}-amd64.tar.gz | tar -vxzf -
+./${system}-amd64/osm version
+```
+
+### For Windows
+Download and unzip the Windows OSM v0.8.0-rc.2 binary via PowerShell:
+```powershell
+wget  https://github.com/openservicemesh/osm/releases/download/v0.8.0-rc.2/osm-v0.8.0-rc.2-windows-amd64.zip -o osm.zip
+unzip osm.zip
+.\windows-amd64\osm.exe version
+```
+
+### Compile from source
+The `osm` CLI can be compiled from source using [this guide](../../install/_index.md).
+
+
+## Installing OSM on Kubernetes
+
+With the `osm` binary downloaded and unzipped, we are ready to install Open Service Mesh on a Kubernetes cluster:
+
+```bash
+osm install
+```
+
+> Note: This document assumes you have already installed credentials for a Kubernetes cluster in ~/.kube/config and `kubectl cluster-info` executes successfully.
+
+This installed OSM Controller in the `osm-system` namespace.
+
+## Deploy Demo Applications
+In this section we will deploy 2 pods in 2 different namespaces:
+- `bookbuyer` - a client, which makes HTTP requests to a `bookstore`
+- `bookstore` - a server, which responds to HTTP requests from `bookbuyer`
+
+
+### Create the Namespaces
+Each application in this demo will reside in a separate namespaces. Create the namespaces with:
+```bash
+kubectl create namespace bookbuyer
+kubectl create namespace bookstore
+```
+
+### Join the new namespaces to the service mesh
+The following `osm` CLI command joins the namespaces to the service mesh:
+
+`//TODO(draychev): link to relevant documentation on how the JOIN command works [#2904]`
+
+```bash
+osm namespace add bookbuyer
+osm namespace add bookstore
```

Comment:
Why not `osm namespace add bookbuyer bookstore` ?

Reply:
I prefer having them on 2 separate lines for clarity - we create 2 namespaces - here are 2 lines. No ambiguity.